### DISCUSSION TITLE
Is this repo legit or does it contains forbidden data ?

### DISCUSSION THREAD

**Discussion by Miminoux** [community/community#42283] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Hi, 
I happened to fall upon this repo by ramdom but it seems quite troublesome : 
https://github.com/gregdm/ecn2014

It does not contain any context so I would say Github should delete the README at least and eventualluy take actions against the owner. 

What do you think ?
Regards,

**Comment by airtower-luna** [comment]
I don't know if the listed people are real, but it sure looks questionable. You can [report the user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam) and ask the GitHub support team to investigate that. Us other users can't take action either way.

**Comment by queenofcorgis** [comment, marked as answer]
Hi @Miminoux 👋🏼  We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our abuse reporting tools. Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms.  I am going to close this post, but for this and any future incidents, please refer to those links.

Thank you!
